addons/udes_stock/static/src/xml/create_planned_transfer_button.xml

```diff
@@ -13,15 +13,9 @@
         </t>
     </t>
 
-    <!--Replace the create button on a picking formview to have 'Planned Transfer' functionality instead of 'Immediate Transfer'-->
+    <!--Remove the create button on a picking form view-->
     <t t-extend="FormView.buttons">
         <t t-jquery="button.o_form_button_create" t-operation="replace">
-            <t t-if="widget and widget.modelName=='stock.picking'">
```

Comment:
Don't we want this if the person is an admin or privilaged user?

Reply:
don't think it's needed in form view, we still have the "create" on list view, and "new planned transfer" on the picking widgets - I see no use for a "Create" button when already looking at a picking! but it's one we can definitely ask to product :)